bbot/presets/nuclei/nuclei-intense.yml

```diff
@@ -0,0 +1,16 @@
+description: Run nuclei scans against all discovered targets, allowing for minimal spidering, with additional discovery modules.
+
+modules:
+  - httpx
+  - nuclei
+  - robots
+  - portfilter
+  - wayback
```

Comment:
If the intention here is to enable modules like wayback and urlscan that emit URLs, we'll need to set `urls=True` on them.